packages/snap/src/ui.ts

```diff
@@ -8,19 +8,23 @@ import { invalidParameters } from './errors';
  * @param origin - Origin of the transaction.
  * @param transaction - Transaction to display.
  * @param selectedNetworkEntrypoint - The selected network entrypoint as a URL. The origin is displayed to the user.
+ * @param publicKey - The selected public key.
  * @returns `true` if the user approves the transaction, `false` otherwise.
  */
 export async function reviewTransaction(
   origin: string,
   transaction: any,
   selectedNetworkEntrypoint: URL,
+  publicKey: string,
 ) {
   const content = panel([
     heading(transactionTitle(transaction)),
     text(`Request from: **${origin}**`),
     divider(),
     ...prettyPrintTx(transaction, text),
     divider(),
+    text(`Selected key: ${publicKey}`),
```

Comment:
Should this say the public key or the name we gave it? eg. "Snap Key 0"

Reply:
I have no opinion about it to be fair.